tests/flow/test_maintain_record_order.py

```
@@ -0,0 +1,65 @@
+from common import *
+
+# tests for record order maintenance
+# eager operations used to reverse record order
+# these test verify that the input record order is maintained on output
+class testMaintainRecordOrder():
+    def __init__(self):
+        self.env, self.db = Env()
```

Ruff (0.7.0)

`Env` may be undefined, or defined from star imports

(F405)